public/Get-DbaReplServer.ps1

```diff
@@ -60,9 +60,14 @@ function Get-DbaRepServer {
         if (Test-FunctionInterrupt) { return }
         foreach ($instance in $SqlInstance) {
             try {
-                # use System.Data instead of Microsoft.Data
-                $sqlconn = New-SqlConnection -SqlInstance $instance -SqlCredential $SqlCredential
-                New-Object Microsoft.SqlServer.Replication.ReplicationServer $sqlconn
+                $server = Connect-DbaInstance -SqlInstance $instance -SqlCredential $SqlCredential
+                $replServer = New-Object Microsoft.SqlServer.Replication.ReplicationServer
+                $replServer.ConnectionContext = $Server.ConnectionContext.SqlConnectionObject
+                $replServer | Add-Member -Type NoteProperty -Name ComputerName -Value $server.ComputerName -Force
+                $replServer | Add-Member -Type NoteProperty -Name InstanceName -Value $server.ServiceName -Force
+                $replServer | Add-Member -Type NoteProperty -Name SqlInstance -Value $server.DomainInstanceName -Force
+
+                Select-DefaultView -InputObject $replServer -Property ComputerName, InstanceName, SqlInstance, DistributorInstalled, DistributorAvailable, IsDistributor, IsPublisher, HasRemotePublisher, DistributionServer, DistributionDatabase, WorkingDirectory, AgentCheckupInterval, DistributionDatabases, DistributionPublishers, ReplicationDatabases, RegisteredSubscribers
             } catch {
                 Stop-Function -Message "Error occurred while establishing connection to $instance" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
```

Comment:
This should have our standard try/catch for the `$server` object. There is no reason to deviate from that for this command.

```powershell
try {
    $server = Connect-DbaInstance -SqlInstance $instance -SqlCredential $SqlCredential
} catch {
    Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
}```